packages/presentation/lib/const/app.dart

```diff
@@ -1,7 +1,12 @@
 abstract class C {
   static const ellipsesInImageButtonCount = 3;
```

Comment:
- I would like to see a better name :)


leaving this comment here as I don't see this file in changes: 
- file: style/text_styles/colors.dart - getter variables
- file: style/text_styles/styles.dart - up to u, you can create a class with static methods in it, in my opinion it's better than this use, this way you can reuse the methods by substituting the color on the UI